test/connect-backbone-to-react.test.js

```diff
@@ -594,6 +594,65 @@ describe('connectBackboneToReact', function() {
     });
   });
 
+  describe('when passed props change to include', function() {
+    let ConnectedTest;
+
+    beforeEach(function() {
+      ConnectedTest = connectBackboneToReact(mapModelsToProps)(TestComponent);
+      wrapper = mount(<ConnectedTest models={modelsMap} />);
+
+      const decoratorUserModel = new UserModel({
+        name: 'Robert',
+        age: '30',
+        hungry: false,
+      });
+
+      const initialModelsMap = {
+        user: userModel,
+        coll: userCollection,
+        decorator: decoratorUserModel,
+      };
+
+      wrapper.setProps({ models: initialModelsMap });
+    });
+
+    afterEach(function() {
+      wrapper.unmount();
+    });
+
+    describe('an undefined model', function() {
+      let setStateSpy;
+      let createListenerSpy;
+      let removeListenerSpy;
+
+      beforeEach(function() {
```

Comment:
@hswolff Is this what you meant by moving the setup to a nested beforeEach?

Reply:
Yep! But let's add come assertions before the describe block to assert that the event listeners are initially bound, bound once, etc. Such that we can then assert they are removed as well. It's a little redundant, but I hope more readable.